bower.json

```diff
@@ -11,7 +11,7 @@
     "gaia-theme": "gaia-components/gaia-theme#0.4.5",
```

Comment:
Might want to generally go through these and update them. I think some of them are getting a bit out of date.